api/v1/dspipeline_types.go

```diff
@@ -66,8 +88,9 @@ type APIServer struct {
 	// +kubebuilder:default:=true
 	// +kubebuilder:validation:Optional
 	EnableRoute bool `json:"enableOauth"`
-	// Include sample pipelines with the deployment of this DSP API Server. Default: true
+	// Include the Iris sample pipeline with the deployment of this DSP API Server. Default: true
 	// +kubebuilder:default:=false
+	// +Deprecated
```

Comment:
This doesn't seem to do anything in the CRD. Could you document that this is deprecated in the CRD description?

Another thought is that since the Iris pipeline isn't a managed pipeline, does this still need to be marked as deprecated?

Reply:
lets just remove this line since we don't need to deprecate it right now